apps/codelab/src/app/codelabs/codelabs-routing.module.ts

```diff
@@ -12,6 +12,10 @@ const routes: Routes = [
     path: 'angular',
     loadChildren: './angular/angular.module#AngularModule'
   },
+  {
+    path: 'angular/30-seconds',
```

Comment:
I guess that a conflict with AngularModule might happen. Architecturally 'angular' route is a route for AngularModule.

To tell the truth, it works, i check. But I am not sure that this approach is a right way to use lazy loading, May be we replace 'angular/30-seconds' to something like 'angular-30-seconds'? or something like that?

Reply:
Yeah, I couldn't find a good solution to this.
It would be very confusing to have it in the folder with other codelabs, this seems like a lesser evil